### PR TITLE
New feature

### PR DESCRIPTION
**Pull request by ctro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # COOLTEST
 
-This is just a test of Git and github!
\ No newline at end of file
+This is just a test of Git and github!
+
+~More~ Better features to this document!
```
Comment (mheadd) [inline README.md]: Should we say `more awesome` features?

Comment (ctro) [reply README.md]: Probably.  I mean, English, right?

Comment (ctro) [reply README.md]: Look better @mheadd ?

Comment (mheadd) [reply README.md]: Yes, approved!

**Comment by ctro** [comment]
Someone plase take care of #2

**Pull request merged**